crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -212,7 +213,7 @@ pub struct ExecutionComponents {
 
 pub struct AuthorityPerEpochStore {
     committee: Arc<Committee>,
-    tables: AuthorityEpochTables,
+    tables: ArcSwapOption<AuthorityEpochTables>,
```

Comment:
I wonder if it worth explaining why we use ArcSwapOption here for future reference.

Reply:
Added comments.